js/client_exporter/xlsx/xlsx_fill_helper.js

```diff
@@ -13,6 +13,30 @@ const xlsxFillHelper = {
         }
         return result;
     },
+    tryCreateFillFromSimpleFormat: function({ backgroundColor, patternStyle, patternColor } = {}) {
+        if(typeUtils.isDefined(backgroundColor) && !(typeUtils.isDefined(patternStyle) && typeUtils.isDefined(patternColor))) {
+            return {
+                patternFill: {
+                    patternType: 'solid',
+                    foregroundColor: {
+                        rgb: backgroundColor
+                    }
+                }
+            };
+        } else if(typeUtils.isDefined(patternStyle) && typeUtils.isDefined(patternColor)) {
```

Comment:
Do we really need this condition? What if the user will set `patternStyle = solid` and `backgroundColor = AAAAAA`?
Both conditions will return `false`, but it is valid property set.

Reply:
Scenario looks like 'solid fill with AAAAA color'.

It is this code:
        onXlsxCellPrepared: e => {
            e.xlsxCell.style.backgroundColor = 'FFFF0000'; // red
            e.xlsxCell.style.patternStyle = 'solid';
        },

which means this OOXML configuration:
                        e.xlsxCell.style.fill = {
                            patternFill: {
                                patternType: 'solid',
                                foregroundColor: { rgb: 'FFFF0000', },
                                backgroundColor: { rgb: undefined, },
                            }
                        };                        

Excel shows it as 'solid red fill'.